### PR TITLE
Update batch of samples to NServiceBus 8 beta 1

### PR DESCRIPTION
**Pull request by mikeminutillo**

### CODE REVIEW

File: samples/serializers/newtonsoft-bson/Newtonsoft_3/Sample/Sample.csproj [hunk 1]
```diff
@@ -5,10 +5,9 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
```
Comment (mikeminutillo) [inline samples/serializers/newtonsoft-bson/Newtonsoft_3/Sample/Sample.csproj:8]: This was the only hitch on this one. This reference causes a compilation error but the sample works without it.

File: samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyReplySagaVersion1.cs [hunk 2]
```diff
@@ -16,8 +16,10 @@ public class MyReplySagaVersion1 :
 
         protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SagaData> mapper)
         {
-            mapper.ConfigureMapping<StartReplySaga>(msg => msg.TheId).ToSaga(saga => saga.TheId);
-            mapper.ConfigureMapping<Reply>(msg => msg.TheId).ToSaga(saga => saga.TheId);
+            mapper.MapSaga(saga => saga.TheId)
+                .ToMessage<StartReplySaga>(msg => msg.TheId)
+                .ToMessage<Reply>(msg => msg.TheId)
+                ;
```
Comment (HEskandari) [inline samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyReplySagaVersion1.cs:22]: Not a big fan of a sole-semi-colon on its own, but I acknowledge that's being subjective (I'm surprised the FXCop didn't catch you doing that!)

**Review by HEskandari — APPROVED** [review]

File: samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyReplySagaVersion1.cs (hunk 2 above)
Comment (mikeminutillo) [reply samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyReplySagaVersion1.cs:22]: Hah. It's habbit. Every time I need to add another mapping to the saga, I need to move the semi-colon so I tend to leave it on it's own for the next one. I don't feel strongly about it.

Comment (timbussmann) [inline samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyReplySagaVersion1.cs:22]:
```suggestion
            mapper.MapSaga(saga => saga.TheId)
                .ToMessage<StartReplySaga>(msg => msg.TheId)
                .ToMessage<Reply>(msg => msg.TheId);
```

File: samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyTimeoutSagaVersion1.cs [hunk 3]
```diff
@@ -16,7 +16,9 @@ public class MyTimeoutSagaVersion1 :
 
         protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SagaData> mapper)
         {
-            mapper.ConfigureMapping<StartTimeoutSaga>(msg => msg.TheId).ToSaga(saga => saga.TheId);
+            mapper.MapSaga(saga => saga.TheId)
+                .ToMessage<StartTimeoutSaga>(msg => msg.TheId)
+                ;
```
Comment (timbussmann) [inline samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyTimeoutSagaVersion1.cs:21]:
```suggestion
            mapper.MapSaga(saga => saga.TheId)
                .ToMessage<StartTimeoutSaga>(msg => msg.TheId);
```

File: samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion2/MyReplySagaVersion2.cs [hunk 4]
```diff
@@ -15,8 +15,10 @@ public class MyReplySagaVersion2 :
 
         protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SagaData> mapper)
         {
-            mapper.ConfigureMapping<StartReplySaga>(msg => msg.TheId).ToSaga(saga => saga.TheId);
-            mapper.ConfigureMapping<Reply>(msg => msg.TheId).ToSaga(saga => saga.TheId);
+            mapper.MapSaga(saga => saga.TheId)
+                .ToMessage<StartReplySaga>(msg => msg.TheId)
+                .ToMessage<Reply>(msg => msg.TheId)
+                ;
```
Comment (timbussmann) [inline samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion2/MyReplySagaVersion2.cs:21]:
```suggestion
            mapper.MapSaga(saga => saga.TheId)
                .ToMessage<StartReplySaga>(msg => msg.TheId)
                .ToMessage<Reply>(msg => msg.TheId);
```

File: samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion2/MyTimeoutSagaVersion2.cs [hunk 5]
```diff
@@ -15,7 +15,9 @@ public class MyTimeoutSagaVersion2 :
 
         protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SagaData> mapper)
         {
-            mapper.ConfigureMapping<StartTimeoutSaga>(msg => msg.TheId).ToSaga(saga => saga.TheId);
+            mapper.MapSaga(saga => saga.TheId)
+                .ToMessage<StartTimeoutSaga>(msg => msg.TheId)
+                ;
```
Comment (timbussmann) [inline samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion2/MyTimeoutSagaVersion2.cs:20]:
```suggestion
            mapper.MapSaga(saga => saga.TheId)
                .ToMessage<StartTimeoutSaga>(msg => msg.TheId);
```

**Review by timbussmann — APPROVED** [review]
pre-approving

**Pull request merged**